compiler/tests/22_pex_func_test.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python2.7
+"""
+Run a regresion test on various srams
+"""
+
+import unittest
+from testutils import header,openram_test
+import sys,os
+sys.path.append(os.path.join(sys.path[0],".."))
+import globals
+from globals import OPTS
+import debug
+
+class sram_func_test(openram_test):
```

Comment:
This should be renamed.

Reply:
Meaning the test?

Reply:
Yes, this is not an "sram_func_test" but a pex test.

Reply:
nvm. I see it